### ISSUE TITLE
The interpolation method for BYOL augmentation is wrong

### ISSUE THREAD

**Issue by songkangsg** [zdaxie/Transformer-Self-Supervised#12]
Under Transformer-SSL/data/build.py, inside the "build_transform" function, under "byol" augmentation type, the interpolation method used in RandomResizedCrop is the default which is BILINEAR, however in the BYOL paper the author used BICUBIC

**Comment by Giles-Billenness** [comment]
Yeah I saw this as well, BILINEAR is less effective as well, although hopefully, it wouldn't have much of an effect.
